mobile/lib/logger.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:get_10101/hybrid_logger.dart';
+import 'package:get_10101/simple_utc_printer.dart';
+import 'package:logger/logger.dart';
+
+Logger get logger => AppLogger.instance;
+
+class AppLogger {
+  static final Logger _logger = Logger(
+      output: HybridOutput(),
+      printer: SimpleUTCPrinter(
+          // Colorful log messages
+          colors: false,
```

Comment:
🙃 why not with colors :)

Reply:
It makes the log text longer and you won't see it anyways in the file afterwards.